templates/experimental/non_opinionated/src/app.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseController } from "@expressots/core";
+import { controller, httpGet, response } from "inversify-express-utils";
+import { Response } from "express";
+import { AppUseCase } from "./app.usecase";
+
+@controller("/")
+class AppController extends BaseController {
+    constructor(private appUseCase: AppUseCase) {
+        super("app-controller");
+    }
+
+    @httpGet("/")
+    execute(@response() res: Response) {
+        return res.send(this.appUseCase.execute());
+    }
+}
+
+export { AppController };
```

Comment:
Should we have an E2E test, that makes a request and asserts the result?
I believe it may be good to provide an example and assert that the setup is correct and HTTP requests are working fine.

Reply:
Agreed, but perhaps as well as in the other comment room for improvement in another issue